opentech/apply/review/models.py

```diff
@@ -26,28 +26,28 @@ def by_reviewers(self):
     def staff_score(self):
         return self.by_staff().score()
 
-    def staff_reccomendation(self):
-        return self.by_staff().reccomendation()
+    def staff_recommendation(self):
```

Comment:
Need to update the template with these changes

Reply:
was there, got missed in a rebase. now fixed